docs/tutorials/intro_tutorial.rst

```diff
@@ -277,6 +273,7 @@ graphics library) to visualize the data in a histogram.
 
 .. code:: python
 
+    # run.py
```

Comment:
lines 276:285 -- as discussed separate this into 2 sections to be explicit for users -- 1 for direct file usages & 1 for jupyter notebook users.